Lib/test/support/__init__.py

```diff
@@ -108,6 +108,18 @@
     "run_with_tz", "PGO", "missing_compiler_executable", "fd_count",
     ]
 
+
+# True if Python is running on Microsoft Windows.
+MS_WINDOWS = (sys.platform == 'win32')
+
+# True if Python is running on Apple macOS.
+MACOS = (sys.platform == 'darwin')
```

Comment:
Likewise here:
```python
MACOS = sys.platform.startswith("darwin")
sys.platform == 'darwin'
```

Reply:
> MACOS = sys.platform.startswith("darwin")

I don't see the purpose of this test. sys.platform is always the "darwin" string on macOS, no?